cohort_4/playbook/ACTIVITIES.md

```diff
@@ -719,6 +815,62 @@ doing one of the following:
   don't want to baby you so we won't remind you.
 - Something else that works best for you.
 
+### Handling Disruptions
```

Comment:
I would like to keep it in the same pull request right now because the fall under the category of disruptions. Also, in the spirit of "closing all of the issues", I would prefer to just merge it in even though it is not perfect. It is 70% there. I prefer not to spend any additional time on this.